core/server/master/src/test/java/alluxio/master/file/meta/options/MountInfoTest.java

```diff
@@ -28,8 +28,9 @@
   public void getFields() {
     AlluxioURI uri = new AlluxioURI("alluxio://localhost:19998/test");
     MountOptions options = MountOptions.defaults();
-    MountInfo info = new MountInfo(uri, options);
+    MountInfo info = new MountInfo(uri, 1, options);
```

Comment:
I thought we should avoid randomness in unit tests, http://www.alluxio.org/docs/master/en/Unit-Testing.html#patterns-to-avoid